app/screens/TalkDetailsScreen.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { ViewStyle } from "react-native"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackParamList } from "../navigators"
+import { Screen, Text } from "../components"
+// import { useNavigation } from "@react-navigation/native"
```

Comment:
[tol] i think its safe to remove all commented code in this file and keeping it clean